pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -2010,55 +2058,7 @@ let
         };
       };
 
-      ms-toolsai.jupyter = callPackage ./ms-toolsai.jupyter { };
-
-      ms-toolsai.jupyter-keymap = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "jupyter-keymap";
-          publisher = "ms-toolsai";
-          version = "1.1.0";
-          sha256 = "sha256-krDtR+ZJiJf1Kxcu5mdXOaSAiJb2bXC1H0XWWviWeMQ=";
-        };
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
-
-      ms-toolsai.jupyter-renderers = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "jupyter-renderers";
-          publisher = "ms-toolsai";
-          version = "1.0.15";
-          sha256 = "sha256-JR6PunvRRTsSqjSGGAn/1t1B+Ia6X0MgqahehcuSNYA=";
-        };
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
-
-      ms-toolsai.vscode-jupyter-cell-tags = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "vscode-jupyter-cell-tags";
-          publisher = "ms-toolsai";
-          version = "0.1.8";
-          sha256 = "sha256-0oPyptnUWL1h/H13SdR+FdgGzVwEpTaK9SCE7BvI/5M=";
-        };
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
-
-      ms-toolsai.vscode-jupyter-slideshow = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "vscode-jupyter-slideshow";
-          publisher = "ms-toolsai";
-          version = "0.1.5";
-          sha256 = "1p6r5vkzvwvxif3wxqi9599vplabzig27fzzz0bx9z0awfglzyi7";
-        };
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
+      ms-vscode-remote.remote-ssh = callPackage ./ms-vscode-remote.remote-ssh { };
```

Comment:
`ms-vscode-remote` comes before `ms-vscode.theme`

Reply:
`ms-vscode` comes before `ms-vscode-remote` when sorting alphabetically because it is shorter.